src/Docker/DatabaseServices.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Docker;
+
+/**
+ * @author  Jesse Rushlow <jr@rushlow.dev>
+ *
+ * @internal
+ */
+final class DatabaseServices
+{
+    public static function getDatabaseSkeleton(string $name, string $version): array
+    {
+        switch (strtolower($name)) {
+            case 'mariadb':
+                return [
+                    'image' => sprintf('mariadb:%s', $version),
+                    'environment' => [
+                        'MYSQL_ROOT_PASSWORD' => 'password',
+                    ],
+                    'ports' => ['3306'],
+                ];
+            case 'mysql':
+                return [
+                    'image' => sprintf('mysql:%s', $version),
+                    'environment' => [
+                        'MYSQL_ROOT_PASSWORD' => 'password',
+                    ],
+                    'ports' => ['3306'],
+                ];
+            case 'postgres':
+                return [
+                    'image' => sprintf('postgres:%s', $version),
+                    'environment' => [
+                        'POSTGRES_PASSWORD' => 'main',
+                        'POSTGRES_USER' => 'main',
+                        'POSTGRES_DB' => 'main',
+                    ],
+                    'ports' => ['5432'],
+                ];
+        }
```

Comment:
Throw an exception down here if none was matched (that will also enforce the `array` return type)

Reply:
done